pkg/provisioner/crdprovisioner.go

```diff
@@ -36,6 +36,7 @@ import (
 	"sigs.k8s.io/azuredisk-csi-driver/pkg/apis/azuredisk/v1alpha1"
 	azDiskClientSet "sigs.k8s.io/azuredisk-csi-driver/pkg/apis/client/clientset/versioned"
 	"sigs.k8s.io/azuredisk-csi-driver/pkg/azureutils"
+	"sigs.k8s.io/azuredisk-csi-driver/pkg/controller"
```

Comment:
Doesnt this create a sort of circular dependency? I see that you need to import this library because of certain constant strings. Might consider moving them to utils from where both `controller` and `provisioner` pkg can make references?

Reply:
It wouldn't create a circular dependency as controllers have no dependency on provisioner package (cloud provisioner interface is defined in controller pkg).

But as you pointed out it would make more sense to move these constants to a different shared package. I will get the changes in soon :)